server/src/lib/clip.ts

```diff
@@ -15,9 +15,15 @@ import Awards from './model/awards';
 import { checkGoalsAfterContribution } from './model/goals';
 import { ChallengeToken, challengeTokens } from 'common';
 
+
+
 const Transcoder = require('stream-transcoder');
 const { Converter } = require('ffmpeg-stream');
 const { Readable } = require('stream');
+const fs_promises = require('node:fs/promises');
+const os = require('node:os');
```

Comment:
Does `import os from 'os';` not work? Similarly does `import { promises } from 'fs';` not work? What are the errors you're getting? Since you're in typescript, you'll have to have the types installed too (`npm install --save @types/node`) - maybe that was why you were getting errors?